io-engine/src/lvs/lvs_store.rs

```diff
@@ -391,10 +393,19 @@ impl Lvs {
         name: &str,
         bdev: &str,
         uuid: Option<String>,
+        cluster_size: Option<u32>,
     ) -> Result<Lvs, Error> {
         let pool_name = name.into_cstring();
         let bdev_name = bdev.into_cstring();
-
+        let cluster_size = if let Some(cluster_size) = cluster_size {
+            if cluster_size % ROUND_TO_MB == 0 {
+                cluster_size
+            } else {
+                0
```

Comment:
An error must be returned in this case, instead of silently ignoring an incorrect size.

Reply:
Code fixed.